lib/core/shared/sdk/embeddedSdk.ts

```diff
@@ -136,6 +163,24 @@ export class EmbeddedSDK extends Kuzzle {
         : options.propagate;
     }
 
+    if (forbiddenEmbeddedActions[request.controller] !== undefined
+      && forbiddenEmbeddedActions[request.controller].has(request.action)
+    ) {
+      throw kerror.get(
+        'api',
+        'process',
+        'forbidden_embedded_sdk_action',
+        request.action,
+        request.controller,
+        ', use user impersonation or security controller instead',
```

Comment:
Why having a custom sentence as a placeholder and not a static one since you are using it here only?

Reply:
Ok I see why now